.gitignore

```diff
@@ -4,6 +4,7 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```

Comment:
Discard this change as it will affect other collaborators on the repo. Just esnure that you dont push your own lock file. A pr changing +15,334 −14,807 cannot be accepted